### ISSUE TITLE
Bug: Scrolling on Playlist/Chapters Menus conflicts with Wheel Commands on input.conf

### ISSUE THREAD

**Issue by Cananbolt** [tomasklaen/uosc#495]
To explain, I have the following keybinds in input.conf,
```
WHEEL_UP add volume -2
WHEEL_Down add volume 2
```
Whenever I scroll through my files in a playlist or chapter in UOSC menus, it would conflict with the keybind above, and it would suddenly activate the WHEEL Keybinds above, causing my sound to increase or decrease, instead of scrolling through the menus.

**Comment by tomasklaen** [comment]
There _has_ been a recent change in handling these events. Are you on master or latest release? Because I can't replicate on either.

**Comment by Cananbolt** [comment]
> There _has_ been a recent change in handling these events. Are you on master or latest release? Because I can't replicate on either.

I completely updated UOSC 4 days ago, following the plethora of updates.

**Comment by tomasklaen** [comment]
Last commit was yesterday. You might have updated while I was in the middle of reworking event handling. Try updating to current master.

But again, I can't replicate, so if the issue still persist, maybe recording a video would tells us more? Because I can't see or think of any issues in code that would cause this.

**Comment by Cananbolt** [comment]
I went ahead and updated the uosc.lua, however, the problem was still there.

Here is a video:
[2023-04-03 10-22-59_00-00-9.033-00-00-38.633-[00.00.000-00.29.738]-audio.webm](https://user-images.githubusercontent.com/100978782/229567092-853278cf-c586-4027-ab32-8c099afc14be.webm)

**Comment by christoph-heinrich** [comment]
Can't reproduce :neutral_face:

**Comment by BhaturaGuy** [comment]
I had same [issue](https://github.com/natural-harmonia-gropius/input-event/issues/34) , just more problematic.
I couldn't scroll randomly while in any menu uosc or any other. 
In my case this was because of some/outdated scripts. I had to update/delete a lot of scripts to make the issue go away.

Maybe try updating the scripts manually, *NOT* using mpv manager to update.
If there aren't any updates within a long time period (1 year or more) consider deleting the script.

This method worked for me, might not work for you so take it with a grain of salt.
Also backup your config directory before trying aforementioned tips.

**Comment by tomasklaen** [comment]
uosc asks mpv to take over the wheel events when it needs them (you hover over the menu). If you keep hovering, and some other script asks for wheel events, it'll overwrite us, and uosc will stop handling them. We can't do much about that. I don't think we can even detect if something like that happens.

**Comment by christoph-heinrich** [comment]
Even if it gets overwritten by some other script, we should get events again once they disable their binding. Instead it does the thing from input.conf like nothing is bound.

**Comment by Cananbolt** [comment]
So, I got rid of every script, besides UOSC and the problem still continued. I also got rid of my script modules, and nothing. 

Edit: Found the issue. It was uosc.conf. That was the only thing I haven't updated. I'll install the latest one and put my configs back and see if the problem still happens.

**Comment by Cananbolt** [comment]
Found the problem, it is `autohide=yes`.
Once I changed it to `autohide=no` it worked again

**Comment by tomasklaen** [comment]
Yeah that was a uosc bug. Thx for catching it. Should be fixed now.

**Comment by Cananbolt** [comment]
Thank you went ahead and tested it and it works correctly. I'm glad I was able to help.
